steps/appointee/appointee-name/AppointeeName.js

```diff
@@ -51,12 +50,11 @@ class AppointeeName extends SaveToDraftStore {
     const title = this.fields.title.value;
     const first = this.fields.firstName.value;
     const last = this.fields.lastName.value;
-    const fullName = parseFullName(`${first} ${last}`, 'all', 1, 0, 0);
     return [
       answer(this, {
         question: this.content.cya.appointeeName.question,
         section: sections.appointeeDetails,
-        answer: decode(`${title} ${fullName.first} ${fullName.last}`)
+        answer: decode(`${title} ${first} ${last}`)
```

Comment:
do we need the decode method? doesn't that change the name?

Reply:
i don't think it changes it but it isn't necessary to have - will remove it